package.json

```diff
@@ -20,6 +19,9 @@
       "eslint --fix",
       "prettier --write"
     ],
+    "*.{json,graphql,gql}": [
+      "prettier --write"
+    ],
     "*.json": [
       "prettier --write"
     ]
```

Comment:
Can you remove this?

Reply:
Hi igorbrasileiro! Thanks for your attention!

> Hey @jsjoaorsouza why not change the message content? Has the same semantic. @victorhmp @icazevedo

You mean, using this doc? https://developers.vtex.com/vtex-developer-docs/docs/storefront-content-internationalization

if yes, i was thinking in a solution where theres no need for graphql, but i will look into it! Thank you very much!